File: w1959653-BackEnd/Booking-BackEnd/booking/booking/src/main/java/lk/ticket/model/event/EventModule.java
package lk.ticket.model.event;

public class EventModule {
    private int eventId;
    private String eventName;
    private String eventLocation;
    private String eventType;
    private String eventDate;
    private String eventTime;
    private int eventNormalTicketPrice;
    private int eventVIPTicketPrice;

    public EventModule() {}

    public String toString(){
        return "EventModule [eventId=" + eventId + ", eventName=" + eventName + ", eventLocation=" + eventLocation + ", eventType=" + eventType + ", eventDate=" + eventDate + ", eventTime=" + eventTime + ", eventNormalTicketPrice=" + eventNormalTicketPrice + ", eventVIPTicketPrice=" + eventVIPTicketPrice + "]";
    }

    public int getEventId() {
        return eventId;
    }

    public void setEventId(int eventId) {
        this.eventId = eventId;
    }

    public String getEventName() {
        return eventName;
    }

    public void setEventName(String eventName) {
        this.eventName = eventName;
    }

    public String getEventLocation() {
        return eventLocation;
    }

    public void setEventLocation(String eventLocation) {
        this.eventLocation = eventLocation;
    }

    public String getEventType() {
        return eventType;
    }

    public void setEventType(String eventType) {
        this.eventType = eventType;
    }

    public String getEventDate() {
        return eventDate;
    }

    public void setEventDate(String eventDate) {
        this.eventDate = eventDate;
    }

    public String getEventTime() {
        return eventTime;
    }

    public void setEventTime(String eventTime) {
        this.eventTime = eventTime;
    }

    public int getEventNormalTicketPrice() {
        return eventNormalTicketPrice;
    }

    public void setEventNormalTicketPrice(int eventNormalTicketPrice) {
        this.eventNormalTicketPrice = eventNormalTicketPrice;
    }

    public int getEventVIPTicketPrice() {
        return eventVIPTicketPrice;
    }

    public void setEventVIPTicketPrice(int eventVIPTicketPrice) {
        this.eventVIPTicketPrice = eventVIPTicketPrice;
    }
}

File: w1959653-FrontEnd/Booking-FrontEnd/src/app/module/event/event-module.ts
import { t } from "@angular/core/src/render3";

export class EventModule {
  eventId: number;
  eventName: string;
  eventLocation: string;
  eventType: string;
  eventDate: string;
  eventTime: string;
  eventNormalTicketPrice: number;
  eventVIPTicketPrice: number;
  eventImage: string;
  buttonName: string;
  systemStatus: string;
  configurationStatus: string;
  availableTickets: number;

  constructor() {
    this.eventId = 0;
    this.eventName = null;
    this.eventLocation = null;
    this.eventType = null;
    this.eventDate = null;
    this.eventTime = null;
    this.eventNormalTicketPrice = 0;
    this.eventVIPTicketPrice = 0;
    this.eventImage = null;
    this.buttonName = null;
    this.availableTickets = 0;
  }
}
